CONTRIBUTING.md

```diff
@@ -29,7 +29,7 @@ We’re especially interested in learning what you need to implement the [Standa
 
 ## Public organizations
 
-We’re looking for public members to help shape the Foundation for Public Code’s long term future. As a member, you’ll have the chance to shape the Foundation’s processes and governance. Find out more about the [benefits and requirements of becoming a member](https://publiccode.net/membership/) and get in touch with us at <membership@publiccode.net>.
+We’re looking for public partners to help shape the Foundation for Public Code’s long term future. Find out more about the [benefits and requirements of becoming a partner](https://publiccode.net/partnership/) and get in touch with us at <partnerships@publiccode.net>.
```

Comment:
This URL doesn't exist yet.

Reply:
Should we remove the link? This kind of language should slowly be phased out of About as well as per #1347

Reply:
The first call to action on publiccode.net links to this contributing file - it's our main doormat for everyone across all of our subdomains. As such, I propose updating rather than deleting.

I do think the request/pitch is slightly different - members get to shape our future, but partners get to benefit from our expertise and connections as we build something together.

Reply:
A, I just realized that `join us` from .net links to this. So it's actually an extremely prominent page, 1 click away from the first view.

With the homepage redesign, there will likely be a new separate page on .net to encourage participation. 
And with the redesign of About, we don't want to pull in people that fast. 
So my sense is that this page contributing file should no longer be that prominent, and that this content is not suited for the contributing for on the About repo.